drivers/clock_control/nrf_power_clock.c

```diff
@@ -228,6 +236,48 @@ static struct clock_control_async_data *list_get(sys_slist_t *list)
 	return async_data;
 }
 
+static void nrf52_anomaly_132_workaround(u32_t start_task)
+{
+	u32_t startup_us;
+
+	if (!IS_ENABLED(CONFIG_NRF52_ANOMALY_132_WORKAROUND)) {
+		return;
+	}
+
+	/* If systick is disabled then it means that there are no conditions
+	 * to apply workaround and startup time measurement (using systick) is
+	 * not started.
+	 */
+	if (!(SysTick->CTRL & SysTick_CTRL_ENABLE_Msk)) {
```

Comment:
SysTick might also be used the timer driver, so you probably should use a build assertion here to check if `CONFIG_CORTEX_M_SYSTICK` is not enabled.